test/vre/chapter31/00-all-fields.e2e.spec.js

```diff
@@ -0,0 +1,116 @@
+const E2eHelpers = require('../../e2e/e2e-helpers');
+const Timeouts = require('../../e2e/timeouts');
+const PageHelpers = require('../../e2e/vre-ch31-helpers');
+const testData = require('./schema/maximal-test.json');
+
+const runTest = E2eHelpers.createE2eTest(
+  (client) => {
+    PageHelpers.initApplicationSubmitMock();
+    PageHelpers.initDocumentUploadMock();
+
+    // renders introduction page
+    client
+      .url(`${E2eHelpers.baseUrl}/employment/vocational-rehab-and-employment/application/chapter31`)
+      .waitForElementVisible('body', Timeouts.normal)
+      .assert.title('Apply for vocational rehabilitation: Vets.gov')
+      .waitForElementVisible('.schemaform-title', Timeouts.slow)
+      .click('.schemaform-start-button');
+
+    E2eHelpers.overrideVetsGovApi(client);
+    E2eHelpers.overrideSmoothScrolling(client);
+    E2eHelpers.expectNavigateAwayFrom(client, '/introduction');
+
+    // Veteran information page
+    client.waitForElementPresent('input[name="root_veteranFullName_first"]', Timeouts.normal);
+    client.assert.cssClassPresent('.progress-bar-segmented div.progress-segment:nth-child(1)', 'progress-segment-complete');
+    PageHelpers.completeVeteranInformation(client, testData.data);
+    client.axeCheck('.main')
```

Comment:
What is this doing?

Reply:
It's running the aXe accessibility checker on the DOM inside the main element.